calcom/cal.com
# Add more variables to event names and a dynamic field

## What does this PR do?



This PR adds the ability to add the host's name, location, and dynamically display the host or attendee's name in the booking title.

Need to approve #2896 as well

Fixes #2443 

https://www.loom.com/share/b5319769799a4c359faa70865bbdbf61

**Environment**: Staging(main branch) / Production

## Type of change





## How should this be tested?



- Choose an event type and change the event name
	- While testing I used "Meeting between {HOST} and {ATTENDEE} at {LOCATION} - dynamic test {HOST/ATTENDEE}"
- Create a booking
- The variables in the booking name should be filled

## Checklist



- I haven't checked if my changes generate no new warnings
- I haven't added tests that prove my fix is effective or that my feature works
- I haven't checked if new and existing unit tests pass locally with my changes
